sympy/physics/mechanics/functions.py

```diff
@@ -422,6 +446,12 @@ def find_dynamicsymbols(expression, exclude=None):
             raise TypeError("exclude kwarg must be iterable")
     else:
         exclude_set = set()
+    if isinstance(expression, Vector):
+        if reference_frame is None:
+            raise ValueError("You must provide reference_frame when passing a "
+                             "vector expression, got %s." % reference_frame")
```

Comment:
`reference_frame` should probably not end with ".

Reply:
yes, that was my mistake. I have rectified it.